test/images/dnsutils/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=alpine
```

Comment:
Thanks again @tianon ;)

@mkumatag please implement that in this PR: armhf => arm32v7, aarch64 => arm64v8